cdap-ui/app/directives/log-viewer/log-viewer.html

```diff
@@ -67,7 +66,7 @@
     Loading Logs...
   </div>
   <!-- LOGS TABLE -->
-  <div ng-if="LogViewer.data.length > 0 && !LogViewer.loading" class="logs-table">
+  <div ng-if="LogViewer.data.length > 0 && !LogViewer.loading" class="logs-table" ng-class="{'table-full-screen': LogViewer.fullScreen}">
```

Comment:
the ng-class here is unnecessary. You already set the full-screen class on the container, you should be able to target the child element from that